src/resolve/resolved-manager.c

```diff
@@ -590,6 +590,7 @@ int manager_new(Manager **ret) {
                 .dnssec_mode = DEFAULT_DNSSEC_MODE,
                 .dns_over_tls_mode = DEFAULT_DNS_OVER_TLS_MODE,
                 .enable_cache = DNS_CACHE_MODE_YES,
+                .cache_from_localhost = false,
```

Comment:
no need to list that here btw. if not specified things are initialized to 0 anyway.

Reply:
Fixed.